android-svc-lib.sh

```diff
@@ -11,6 +11,13 @@ export g_green=$(printf '%s\n' '\033[0;32m' | sed -e 's/[\/&]/\\&/g')
 export g_yellow=$(printf '%s\n' '\033[0;33m' | sed -e 's/[\/&]/\\&/g')
 export g_nc=$(printf '%s\n' '\033[0m' | sed -e 's/[\/&]/\\&/g') # no color
 
+###
+# init g_repoUrl value
+# GLOBALS:
+#  l_targetDir
+#  g_repoUrl will be filled by this method, and contains an url from https://github.com/aosp-mirror/platform_frameworks_base repository
```

Comment:
This depends on your Android ROM. For LineageOS this repository will not be used. Instead the official LineageOS repository will be used. Support for more ROMs will be added in the future.